packages/gatsby-theme-api-docs/src/components/description.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { designSystem, Markdown } from '@commercetools-docs/ui-kit';
+import { markdownFragmentToReact } from '@commercetools-docs/gatsby-theme-docs';
+
+const DescriptionContainer = styled.div`
+  max-width: ${designSystem.dimensions.widths.pageContent};
+`;
+
+const Description = (props) => (
+  <DescriptionContainer>
+    {typeof props.children === 'string' ? (
+      <Markdown.TypographyContainer>
+        {markdownFragmentToReact(props.children)}
+      </Markdown.TypographyContainer>
+    ) : (
+      props.children
+    )}
+  </DescriptionContainer>
+);
+Description.propTypes = {
+  children: PropTypes.any,
```

Comment:
```suggestion
  children: PropTypes.node,
```

Reply:
I chose any because it can be a string or a node.  But maybe I'm wrong here and a string would go through as a node.